src/components/Post/Post.tsx

```diff
@@ -0,0 +1,15 @@
+import Post from "./Post.types.ts";
+import { Anchor } from "@mantine/core";
+import { useParams } from "react-router-dom";
+
+export function PostItem(props: Post) {
```

Comment:
destructure props, it makes its more readable and makes it easier to understand which props a component is accepting.
```suggestion
export function PostItem({id, text}: Post) {
```